@here/harp-mapview/lib/ScreenProjector.ts

```diff
@@ -69,7 +70,7 @@ export class ScreenProjector {
      * outside the near / far plane.
      */
     project3(
-        source: THREE.Vector3,
+        source: THREE.Vector3 | Vector3Like,
```

Comment:
No need for THREE.Vector3 b/c it is a Vector3Like

Reply:
See: 0716eb78a39efe80daee449146d5911b526e8dfb